engine/src/test/java/org/hawkular/alerts/engine/RulesEngineTest.java

```diff
@@ -179,11 +183,11 @@ public void thresholdTest() {
         assertEquals(e.toString(), 1, e.getConditionSetSize());
         assertEquals("trigger-2", e.getTriggerId());
         assertTrue(e.isMatch());
-        assertTrue(!v.equals(e.getValue()));
+        assertFalse(v.equals(e.getValue()));
         v = e.getValue();
         assertTrue(e.toString(), v.equals(5.0D) || v.equals(10.0D));
         assertEquals(e.getCondition().toString(), "NumericData-01", e.getCondition().getDataId());
-        assertTrue(e.getCondition().getDataId().equals("NumericData-01"));
+        assertEquals("NumericData-01", e.getCondition().getDataId());
```

Comment:
_I_ would prefer to have such refactorings in a separate PR

Reply:
Me too usually. Last time I opened a new PR because of refactorings like that, @josejulio said I should add it in an existing one :-D

Reply:
> > Uh no :-D I spent one day of merging already.
> 
> Don't worry, I'll take care of it smiley

Ok :) If you need help, don't hesitate to raise your hand. I could help as well